functions.md

```diff
@@ -421,6 +438,7 @@ from-json("[1,2]")       => [1, 2]
 from-json("[1,2", "BAD") => "BAD"
 from-json("[1,2")        => error
 from-json(null)          => null
+from-json(null)          => null
```

Comment:
This line is a duplicate?

Reply:
Yes, I didn't mean to add that line. I'll remove it